internal/descriptor/provider/provider_test.go

```diff
@@ -0,0 +1,50 @@
+package provider_test
+
+import (
+	"testing"
+
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"github.com/kyma-project/lifecycle-manager/internal/descriptor/cache"
+	"github.com/kyma-project/lifecycle-manager/internal/descriptor/provider"
+	"github.com/kyma-project/lifecycle-manager/pkg/testutils/builder"
+)
+
+func TestGetDescriptor_OnEmptyCache_ReturnsParsedDescriptor(t *testing.T) {
+	descriptorCache := cache.NewDescriptorCache()
+	sut := provider.NewCachedDescriptorProvider(descriptorCache)
+	template := builder.NewModuleTemplateBuilder().Build()
+	_, err := sut.GetDescriptor(template)
```

Comment:
For me personally it is harder to read `sut.GetDescriptor(template)`. I start wondering what is actually is the System Under Test.

How about: 
```suggestion
	_, err := descriptorProvider.GetDescriptor(template)
```
?


And an empty line above is needed.